config/development.yaml

```diff
@@ -6,7 +6,7 @@ server:
 # Database credentials
 database:
   user_name: "root"
-  password: "password"
+  password: "Ayu@4998"
```

Comment:
Add development yaml file in gitignore as this is specific for individual developer.